### PR TITLE
Add Planq Network

### PR DESCRIPTION
**Pull request by ivanshukhov**

### CODE REVIEW

File: packages/web/config/price.ts [hunk 1]
```diff
@@ -1064,4 +1064,14 @@ export const PoolPriceRoutes: IntermediateRoute[] = [
     spotPriceDestDenom: "uosmo",
     destCoinId: "pool:uosmo",
   },
+  {
+    alternativeCoinId: "pool:aplanq",
+    poolId: "898",
+    spotPriceSourceDenom: DenomHelper.ibcDenom(
+      [{ portId: "transfer", channelId: "channel-492" }],
```
Comment (JeremyParish69) [inline packages/web/config/price.ts:1071]: can you please register the ibc connection with channel-492 to Chain Registry/_IBC ?

Comment (ivanshukhov) [reply packages/web/config/price.ts:1071]: Thanks for reviewing, the connection is already in there though: 
https://github.com/cosmos/chain-registry/blob/master/_IBC/osmosis-planq.json

**Comment by JeremyParish69** [comment]
Can you please verify ibc transfers via the staging link?: https://osmosis-frontier-git-fork-planq-network-add-planq-osmo-labs.vercel.app/

File: packages/web/config/price.ts (hunk 1 above)
Comment (ivanshukhov) [reply packages/web/config/price.ts:1071]: > Can you please verify ibc transfers via the staging link?: https://osmosis-frontier-git-fork-planq-network-add-planq-osmo-labs.vercel.app/

Can verify deposit and withdraw works:
https://www.mintscan.io/osmosis/txs/D376FA2081E7B7A27C6A32D03579B132B990EA3AF81DB2831759870B376B2414
https://ping.pub/planq/tx/7C37E0D19FB1AD040785D51D53183BE32F1AE91C6B081C01F9D640A4B3C3E709

Comment (JeremyParish69) [reply packages/web/config/price.ts:1071]: oh, opps I missed it, sorry. Thanks for confirming that

**Review by JeremyParish69 — APPROVED** [review]
LGTM

**Comment by JeremyParish69** [comment]
Please address the conflicts

**Comment by ivanshukhov** [comment]
Should be resolved

**Comment by JeremyParish69** [comment]
> Should be resolved

Sorry, looks like there are still conflicts

**Comment by ivanshukhov** [comment]
> > Should be resolved
> 
> Sorry, looks like there are still conflicts

sorry, now.

**Pull request merged**